.storybook/preview.tsx

```diff
@@ -169,3 +170,14 @@ export const parameters = {
     viewports: INITIAL_VIEWPORTS,
   },
 };
+
+global.___loader = {
+  enqueue: () => {},
+  hovering: () => {},
+};
+
+global.__BASE_PATH__ = '/';
+
+window.___navigate = (pathname) => {
+  action('NavigateTo:')(pathname);
+};
```

Comment:
Question: assuming these are all Gatsby Link patching?

Reply:
Yup can add a comment in there.